pages/tutorials/getting_started.md.erb

```diff
@@ -8,33 +8,39 @@ This guide will help you setup your first Buildkite pipeline. If you don’t hav
 
 [Buildkite agents](/docs/agent/v3/) are small, reliable and cross-platform build runners that run automated builds. Buildkite never accesses your code, and does not run any agents, so you need to install and run agents on your own infrastructure. You can do this on your local development machine, an existing CI machine, or a new server.
 
-1. Follow the [Agent installation instructions](/docs/agent/v3/installation) to install and start an agent
-
-2. Make sure you’ve configured the agent token, as that is what connects the agent to your Buildkite account. Refer to [Buildkite Agent Configuration](/docs/agent/v3/configuration) for more information.
+1. Follow the [Agent installation instructions](/docs/agent/v3/installation) to install and start an agent.
+2. Make sure you've configured the agent token, as that is what connects the agent to your Buildkite account. Refer to [Buildkite Agent Configuration](/docs/agent/v3/configuration) for more information.
+3. Opt in to the YAML steps editor. This is the newer way of configuring pipelines. Refer to [Pipeline Upgrade - Using YAML Steps for new pipelines](/docs/tutorials/pipeline-upgrade#using-yaml-steps-for-new-pipelines) for more information.
 
 ## Add the sample pipeline
 
-[Pipelines](/docs/pipelines/) are how Buildkite represents your CI workflow, a series of steps that will be run on your code.
-
-We’ve created a [bash-example test repository](https://github.com/buildkite/bash-example) to use as your first pipeline. Click the “Add to Buildkite” button below to open a new pipeline page for the example pipeline:
-
-<a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/bash-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add Bash Example to Buildkite" class="no-decoration" width="160" height="30"></a>
+[Pipelines](/docs/pipelines/) are how Buildkite represents your CI workflow, a series of steps that you run on your code.
 
-Accept the pre-filled defaults, and click *Create Pipeline*.
+Choose a sample pipeline to use as your first pipeline:
 
-## Run your first build
+1. Click the appropriate *Add to Buildkite* button:
 
-Now that you’re running an agent and have created a pipeline you’re ready to run your first build. Click the *New Build* button in the top right corner:
+    [bash-example test repository](https://github.com/buildkite/bash-example) 
```

Comment:
Do you think we need an explicit if you're running a Windows agent, or if you're running anything else type text?

Reply:
The steps above mention they need to start the agent. I've added a bit to that section to tell them how to confirm the agent is running and linked to their org correctly.

This is universal, whatever they're on. In theory, they could use the bash example repo on windows, IF they can point to git bash successfully (which admittedly I could not) . . .